src/components/sbb-toggle-check/sbb-toggle-check.tsx

```diff
@@ -80,9 +94,13 @@ export class SbbToggleCheck implements AccessibilityProperties {
           aria-labelledby={this.accessibilityLabelledby}
         />
         <span class="toggle-check__container">
-          <span class="toggle-check__label">
-            <slot />
-          </span>
+          {this._hasLabelText ? (
+            <span class="toggle-check__label">
+              <slot onSlotchange={(event): void => this._onLabelSlotChange(event)} />
+            </span>
+          ) : (
+            <slot onSlotchange={(event): void => this._onLabelSlotChange(event)} />
+          )}
```

Comment:
what's the purpose of the alternative here? I think we can live without?

Reply:
The 'sbb-toggle-check__container' has a gap rule; if the span is in the DOM, rule is applied, even if the nested slot is empty, and this causes the unwanted padding.